packages/blade/src/components/Spinner/BaseSpinner/SpinningBox.web.tsx

```diff
@@ -0,0 +1,23 @@
+import styled, { css, keyframes } from 'styled-components';
+import { motion as spinnerMotion } from './spinnerTokens';
+import Box from '~components/Box';
+import { getIn, makeMotionTime } from '~utils';
+
+const rotate = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+`;
+
+const SpinningBox = styled(Box)(({ theme }) => {
```

Comment:
I think this should be inline element, `inline-block` or `inline-flex` to help with styling other nearby content, thoughts?

Reply:
Yeah makes sense. Made it an `inline-flex`